README.md

```diff
@@ -31,6 +31,8 @@ on:
 
 jobs:
   CLABot:
+    # Skip job when triggered by an issue comment rather than a PR comment
+    if: github.event_name == 'pull_request_target' || contains(github.event.comment.html_url, '/pull/')
```

Comment:
Oops - this reference to `pull_request_target` only makes sense _after_ #13 is merged. I'll switch this back to `pull_request` in the meantime.

```suggestion
    if: github.event_name == 'pull_request' || contains(github.event.comment.html_url, '/pull/')
```